test/manual/amp-analytics-visible-repeat.amp.html

```diff
@@ -56,6 +56,23 @@
       }
     </script>
   </amp-analytics>
+  <amp-analytics>
```

Comment:
Please keep manual test focus on one feature and move it to either the selector example, or the example of itself. 
Also we have the test/manual/amp-analytics folder.